doc/tutorials/dune-package-management/setup.md

```diff
@@ -0,0 +1,197 @@
+# Setting up Package Management with Dune
+
+The idea of package management with Dune has been to be as unobtrusive as
+possible. Thus most projects can easily be built with just the minimum of
+changes.
+
+In this tutorial we'll create a simple project to use the OCaml package
+management for the very first time.
```

Comment:
In the goal of reproducibility, we should make it clear how to get a dune binary that can be used in this tutorial (I assume it doesn't work with a released version), or at least clarify that it doesn't work with the dune that people have on their machines.